links/index.html

```diff
@@ -19,6 +19,6 @@
+    <script>var links={"/links?home":"https://maskbook.com","/links?mailchimp-signup-maskbook":"http://eepurl.com/gmNnlb","/links?github-forum":"MASKBOOK-Talks","/links?report-bug":"MASKBOOK/issues/new?template=bug.md&assignees=jack-works&projects=DimensionDev/Maskbook/3&title=%5BBug%5D%20&labels=Type%3A%20Bug","/links?ios-app-store":"https://apps.apple.com/app/id1478382964","/links?google-play":"https://play.google.com/store/apps/details?id=com.dimension.maskbook","/links?chrome":"https://chrome.google.com/webstore/detail/jkoeaghipilijlahjplgbfiocjhldnap","/links?firefox":"https://addons.mozilla.org/firefox/addon/maskbook","/brand-guide":"https://dimensiondev.github.io/Maskbook-VI","/faq?1":"MASKBOOK/wiki/FAQ#why-is-there-a-strange-link-in-my-tweet","/faq?2":"MASKBOOK/wiki/FAQ#why-does-maskbook-recommend-users-to-publish-their-public-keys-in-their-bio","/faqcc?1":"MASKBOOK/wiki/FAQ:-Cryptocurrency-Features#why-is-the-total-collected-amount-slightly-less-than-the-total-spent-amount","/download-latest":"MASKBOOK/releases/latest/download/Maskbook.base.zip","/news":"https://news.maskbook.com","/install-dev":"MASKBOOK/wiki/Manual-installation"};if(/^\/install/g.test(location.pathname)){var agent=navigator.userAgent;/Chrome/i.test(agent)?location.replace(links["/links?chrome"]):/Firefox/i.test(agent)?location.replace(links["/links?firefox"]):location.replace("/download-links")}else{var path=location.pathname.replace(/\/$/g,""),link=links[path+location.search];void 0!==link?(link=link.replace(/^MASKBOOK/,"https://github.com/DimensionDev/Maskbook"),document.links.namedItem("click").href=link,location.replace(link+location.hash)):document.body.className="link-not-found"}</script>
```

Comment:
It's unreadable.

Reply:
Maybe we should avoid minimizing this part. 🤔

Reply:
The source is `/links/main.js` via `npm run-script build:links`